libs/libmbim/Makefile

```diff
@@ -8,13 +8,13 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=libmbim
-PKG_VERSION:=1.30.0
-PKG_RELEASE:=2
+PKG_VERSION:=1.32.0
+PKG_RELEASE:=3
```

Comment:
You should reset `PKG_RELEASE` to 1 when bumping a version.

Reply:
Got it, thanks!